src/openvic-simulation/history/DiplomaticHistory.cpp

```diff
@@ -84,61 +103,86 @@ bool DiplomaticHistoryManager::load_diplomacy_history_file(CountryManager const&
 		"alliance", ZERO_OR_MORE, [this, &country_manager](ast::NodeCPtr node) -> bool {
 			Country const* first;
 			Country const* second;
```

Comment:
```suggestion
			Country const* first = nullptr;
			Country const* second = nullptr;
```
Can you null-initialise these here and in call cases below (with MSVC they'll have random addresses if they're not explicitly initialised, and even though we expect a first and second country we still need to account for the possibility that they're not in the defines).

Reply:
Good to know, will do.